pkgs/development/python-modules/rapidfuzz-capi/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+}:
+
+buildPythonPackage rec {
+  pname = "rapidfuzz-capi";
+  version = "1.0.2";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "maxbachmann";
+    repo = "rapidfuzz_capi";
+    rev = "v.${version}";
+    sha256 = "127mwjbyp2nfj846bxb3aajmhsgzfx4wfvizpnwb30yv2fcrpy7l";
+  };
+
+  pythonImportsCheck = [ "rapidfuzz_capi" ];
+
+  meta = with lib; {
+    description = "C-API of RapidFuzz, which can be used to extend RapidFuzz from separate packages";
+    homepage = "https://github.com/maxbachmann/rapidfuzz_capi";
+    license = licenses.mit;
+    maintainers = with maintainers; [ dotlambda ];
```

Comment:
nope

Reply:
@dotlambda ? https://github.com/NixOS/nixpkgs/pull/168580/commits/28b65c3cb02f91a37c71eb4baf76acf480fde848#diff-7051ba4a27a1aa6bb264875507a22237c6aaf3566a13335c6c96f76593be5314

i'm probably going to remove this file and wait for your pr to get merged

Reply:
Oops, I meant to put this on the respective line in rapidfuzz-cpp